### PR TITLE
highs: add version 1.5.3

### PR DESCRIPTION
**Pull request by hedtke**
Specify library name and version:  **highs/1.5.3**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/highs/all/conanfile.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def build(self):
         apply_conandata_patches(self)
         cmake = CMake(self)
         cmake.configure()
-        cmake.build(target="libhighs")
+        cmake.build(target="highs")
```
Comment (RubenRBS) [inline recipes/highs/all/conanfile.py:68]: I'm guessing the target changed between versions? If the old target is correct for the old one, this should be conditional on the current version, no?

**Review by RubenRBS — COMMENTED** [review]
Hi! Thanks a lot for your contribution, we appreciate it :) I have one minor question regarding the target change

File: recipes/highs/all/conanfile.py (hunk 1 above)
Comment (hedtke) [reply recipes/highs/all/conanfile.py:68]: Yes, target name changed. The new name also works with the old version :-)

Comment (RubenRBS) [reply recipes/highs/all/conanfile.py:68]: Oh, that's great then, thanks for clarifying!

**Review by SSE4 — APPROVED** [review]

**Review by jcar87 — APPROVED** [review]

**Pull request merged**